### DISCUSSION TITLE
math: \newcommand

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#17167] [Other Feature Feedback, Questions, & Ideas]
For a brief moment in time after its release, the math feature supported `\newcommand`.

Screenshot:

![screenshot](https://user-images.githubusercontent.com/181628/169795447-5cce11b1-717b-4dd7-b5ca-693033009183.png)

This doesn't work anymore, code:

$$
\newcommand\myexp[1]{e^{#1}}
$$

Inline: $\myexp{i}$

Display: 

$$
\myexp{i}
$$

**Comment by hrbigelow** [comment]
I'd definitely like to see this as well!

**Comment by ricmua** [comment]
Same
